CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.1.0...HEAD)
+
+## [0.1.0](https://github.com/OpenLineage/OpenLineage/releases/tag/0.1.0) - 2021-08-11
+
+* OpenLineage initial public release.
```

Comment:
I would list:
- the integrations
  - Apache Airflow: 
    - BigQuery 
    - Postgres
    - Snowflake
    - Redshift
    - Great Expectations
  - Apache Spark
  - dbt
- the initial spec: and its core facets: https://github.com/OpenLineage/OpenLineage/blob/6fdb68bc56cf6257c0e94cddf3fb9f215473fec8/spec/OpenLineage.md
- the clients: java/python

Reply:
Yeah, good idea. I can add a bit more detail here about the initial release 👍